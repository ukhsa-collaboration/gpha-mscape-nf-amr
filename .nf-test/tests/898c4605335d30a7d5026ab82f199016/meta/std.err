[33mNextflow 25.04.8 is available - Please consider updating your version to it(B[m
