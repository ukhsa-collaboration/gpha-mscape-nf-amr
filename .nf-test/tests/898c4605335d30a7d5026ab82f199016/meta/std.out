Process `GZ_TO_FASTQ` declares 1 input channel but 0 were specified

 -- Check script '/home/jovyan/git/gpha-mscape-nf-amr/.nf-test-898c4605335d30a7d5026ab82f199016.nf' at line: 38 or see '/home/jovyan/git/gpha-mscape-nf-amr/.nf-test/tests/898c4605335d30a7d5026ab82f199016/meta/nextflow.log' file for more details
